ElectronClient/app/gui/Header.jsx

```diff
@@ -161,11 +161,13 @@ class HeaderComponent extends React.Component {
 		const inputStyle = {
 			display: 'flex',
 			flex: 1,
+			marginLeft: 20,
```

Comment:
I think it looks better with `marginLeft: 15,` but in the end @laurent22 will have to decide.

Reply:
I agree. At some point, we might look at improving the look of the toolbar, but for now lets keep margins as they are. So please @SubodhDahal could you remove this change? Otherwise it's good to merge.

Reply:
@laurent22 I did some testing and `marginLeft: 15` is still too wide. Without a margin the field looks stuck to the button, so it looks best with `marginLeft: 10`.

Reply:
Ok fair enough, let's go with a 10px left margin then.

Reply:
I've now updated the left margin to be 10px.